### DISCUSSION TITLE
Big Thanks for 2021

### DISCUSSION THREAD

**Discussion by frosencrantz** [saulpw/visidata#1282] [General]
I've been meaning to write this since the end of last year.  I wanted to shout out a big thanks to @saulpw and @anjakefala for another great year on VisiData. You are both available to help make VisiData better, making sure bugs are fixed,  questions are answered, and improvements are pondered.

It looks like there were an amazing 11 releases including some quick point releases to fix community reported issues.

There was a live VisiData class at StrangeLoop.  

Also, I want to thank the community for helping out like the code or support such as @geekscrapy @ajkerrigan, @aborruso, @cwarden, @hoclun-rigsep, @SuRaMoN, @pacien, and a few more (sorry for those I missed).

It has been really great to see some corporate sponsors [October Swimmer](https://www.octoberswimmer.com/) and [General UI](https://genui.com),

I do hope that we see more of the community contributing either with discussions, code, docs, sponsorship, etc.

Thanks  Saul and Anja and the community.  I hope this will be another great year for VisiData.

**Comment by aborruso** [comment]
@frosencrantz I want to thank you, you do an amazing job here in this community.

This space is wonderful, for the quality and the ideas of the people. 

So I'm with you and add a very special thanks to @saulpw and @anjakefala

**Comment by saulpw** [comment]
Thanks so much, @frosencrantz!  It was a busy year for sure.  I apologize for being slow or sporadic to respond to issues, but it looks like 2022 will be even busier, so please be patient.  Some interesting things are in the pipeline, though.  Stay tuned =)
